include/swift/Subsystems.h

```diff
@@ -242,10 +243,10 @@ namespace swift {
   performSILGeneration(FileUnit &SF, SILOptions &options,
                        Optional<unsigned> StartElem = None);
 
-  using ModuleOrSourceFile = PointerUnion<ModuleDecl *, SourceFile *>;
-
-  /// Serializes a module or single source file to the given output file.
-  void serialize(ModuleOrSourceFile DC, const SerializationOptions &options,
+  /// Serializes a module and zero or more source files to the given output file.
```

Comment:
This isn't right. We still need one swiftmodule per input file, or incremential compilation won't work.